astro/src/content/docs/release-notes/index.mdx

```diff
@@ -46,6 +46,10 @@ Looking for release notes older than 1.23.0? Look in the [release notes archive]
 
 <ReleaseNoteHeading version='1.53.3' releaseDate='October 25, 2024'/>
 
+<ThemeUpdateWarning version="1.53.3">
+  A fix for FusionAuth SSO session management with external identity providers requires a change to Google IdP usage.
```

Comment:
I think that a link to the look-and-feel upgrade page would be good here

Reply:
That is handled by the `ThemeUpdateWarning` components.